### ISSUE TITLE
Kerberos supprt

### ISSUE THREAD

**Issue by Natureshadow** [django-ldapdb/django-ldapdb#174]
Currently, djangoldapdb only supports anonymous and non-anonymous simple binds. The underlying python-ldap does support Kerberos (SASL GSSAPI) so we should do so, too.

There are at least two scenarios:

 1. The webserver hvaing a static Kerberos ticket
 2. The browser passing a Kerberos ticket through the webserver to Django

Both cases should be supported

**Comment by rbarrois** [comment]
I agree. The first case should be relatively easy; but the second would be much more complex.

However, that second approach might make sense for other DBs as well — one might want to have a connection to a PostgreSQL DB authenticated with the actual user's credentials.
